sklearn/metrics/classification.py

```diff
@@ -1063,7 +1063,7 @@ def precision_recall_fscore_support(y_true, y_pred, beta=1.0, labels=None,
 
         # All labels are index integers for multilabel.
         # Select labels:
-        if not np.all(labels == present_labels):
+        if not np.all(labels == present_labels) & n_labels < len(labels):
```

Comment:
I don't think that's the right condition (perhaps `or n_labels != len(...)`), and it's certainly a misuse of `&`